installation/CanvasVault/setup_CanvasVault.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+
+export AUTH_PATH="jwt-k8s-sman"
+export VAULT_DEV_ROOT_TOKEN_ID="egalegal"
+# optional create istio ingress virtualservice when hostname is set
+export CANVASVAULT_VS_HOSTNAME="canvas-vault-hc.ihc-dt.cluster-3.de"
```

Comment:
Check the host name

Reply:
hostname is correct:

![grafik](https://github.com/tmforum-oda/oda-canvas/assets/11094156/e29a3ad5-6b1a-42aa-93e9-4a8813348003)